lxd/instance/drivers/driver_qemu.go

```diff
@@ -85,6 +85,9 @@ const qemuDeviceIDPrefix = "dev-lxd_"
 // qemuNetDevIDPrefix used as part of the name given QEMU netdevs generated from user added devices.
 const qemuNetDevIDPrefix = "lxd_"
 
+// qemuMinimumUSBPorts is the minimum amount of USB ports for VMs.
+const qemuMinimumUSBPorts = 4
```

Comment:
Would it be better calling these spare ports? So they indicate they are for accommodating hotplugged devices (when we support that)?

Reply:
Could you also update this https://github.com/lxc/lxd/blob/master/lxd/instance/drivers/driver_qemu.go#L2607-L2610 to use the constant (and so name the constant generically)?